content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -62,7 +62,7 @@ kubectl config view
 # get the password for the e2e user
 kubectl config view -o jsonpath='{.users[?(@.name == "e2e")].user.password}'
 
-kubectl config view -o jsonpath='{.users[].name}'    # get a list of users
```

Comment:
@DanyC97 guess you can confirm this as you added the line in #13257 :)

Reply:
Actually, `.users[].name` should list all users, not just the first user. Are you sure that you don't just have a single user on the system?

Reply:
I have multiple users in `~/.kube.config`:

```yaml
apiVersion: v1
clusters:
current-context: snow-cluster
kind: Config
preferences: {}
users:
- name: foo
  user:
    token: abc123
- name: bar
  user:
    token: xyz456
```

`kubectl config view -o jsonpath='{.users[].name}'` outputs `bar`

`kubectl config view -o jsonpath='{.users[0].name}'` also outputs `bar`

`kubectl config view -o jsonpath='{.users[1].name}'` outputs `foo`

Reply:
<img width="820" alt="Screen Shot 2019-08-20 at 9 58 51 AM" src="https://user-images.githubusercontent.com/1571918/63367634-27335480-c331-11e9-929f-f3428dbd3d63.png">

Reply:
hmm, interesting. Perhaps this is a difference between `jq` and the jsonpath flag for `kubectl`

Reply:
@jaypipes I hope this illustrates that `get a list of users` may not be the best description of the command

Reply:
Indeed it does! :)